### PR TITLE
Job Result section

### PR DESCRIPTION
**Pull request by Mykhailenko**

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -826,6 +827,37 @@ result = file.getBytes()
 resultMetadata.put("content.type","image/png")
 ----
 
+=== Job Results
+
+Job Results are composed of two things: *Job Result List* and *Job Result Map*.
```
Comment (mcompastie) [inline src/docs/user/ProActiveUserGuide.adoc]: `Job Results are composed of two elements`

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -826,6 +827,37 @@ result = file.getBytes()
...
+Each task can contribute as for *Job Result List* as for *Job Result Map*.
+After job is finished, you can see Job Results in the Scheduler portal:
+
+image::scheduler-result-map.png[align=center]
+
+==== Job Result List
+
+Each task has its own result. However, some task results can be marked as *Task Result Added to Job Result*. In that case, this task result will be added to the *Job Result List*.
```
Comment (mcompastie) [inline src/docs/user/ProActiveUserGuide.adoc]: `Each task has its own results`

File: src/docs/user/ProActiveUserGuide.adoc [hunk 3]
```diff
@@ -826,6 +827,37 @@ result = file.getBytes()
...
+
+In the Job xml schema it is available as `preciousResult`:
+
+``` {.xml}
+<task name="Init_Task" preciousResult="true" >
+```
+
+==== Job Result Map
+
+Job has key-value map associated to it, which is called *Job Result Map*. Each task can contribute to this *Job Result Map*. Consider groovy example:
```
Comment (mcompastie) [inline src/docs/user/ProActiveUserGuide.adoc]: `Consider the following groovy example:`

File: src/docs/user/ProActiveUserGuide.adoc (hunk 2 above)
Comment (Mykhailenko) [reply src/docs/user/ProActiveUserGuide.adoc]: i believe task can have only single result. so I keep it singular

File: src/docs/user/ProActiveUserGuide.adoc (hunk 1 above)
Comment (Mykhailenko) [reply src/docs/user/ProActiveUserGuide.adoc]: thanks

File: src/docs/user/ProActiveUserGuide.adoc (hunk 3 above)
Comment (Mykhailenko) [reply src/docs/user/ProActiveUserGuide.adoc]: thanks

File: src/docs/user/ProActiveUserGuide.adoc [hunk 4]
```diff
@@ -826,6 +827,37 @@ result = file.getBytes()
 resultMetadata.put("content.type","image/png")
 ----
 
+=== Job Results
+
+Job Results are composed of two things: *Job Result List* and *Job Result Map*.
+Each task can contribute as for *Job Result List* as for *Job Result Map*.
+After job is finished, you can see Job Results in the Scheduler portal:
```
[the remark this answers is not in the document]
Comment (Mykhailenko) [reply src/docs/user/ProActiveUserGuide.adoc]: thanks

File: src/docs/user/ProActiveUserGuide.adoc (hunk 2 above)
Comment (mcompastie) [reply src/docs/user/ProActiveUserGuide.adoc]: Ok

**Pull request merged**